Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -482,15 +482,15 @@ def rasterize_command(urls: Union[list[str], str], rasterize_timeout: int) -> li
         }
     )
     demisto.debug(f'Rasterize Data: {res_rasterize}')
-    return [res['Contents'] for res in res_rasterize]
+    return [res['Contents'] for res in res_rasterize][:len(urls)]
```

Comment:
how do we know the first x are the relevant ones?

Reply:
It's the same as when we use `response[0]` for a `demisto.executeCommand` output across the repo. Any further additions to the response by the server is ignored.